x-pack/plugins/monitoring/public/components/kibana/instances/instances.js

```diff
@@ -0,0 +1,168 @@
+        <div
+          title={`Instance status: ${status}`}
+          className="monTableCell__status"
+        >
+          <KibanaStatusIcon status={status} availability={kibana.availability} />&nbsp;
+          { !kibana.availability ? (
+            <FormattedMessage
+              id="xpack.monitoring.kibana.listing.instanceStatus.offlineLabel"
+              defaultMessage="Offline"
+            />
+          ) : capitalize(status) }
+        </div>
+      )
+    },
+    {
+      name: i18n.translate('xpack.monitoring.kibana.listing.loadAverageColumnTitle', {
+        defaultMessage: 'Load Average'
+      }),
+      field: 'os.load.1m',
+      render: value => (
+        <span>
+          {formatMetric(value, '0.00')}
+        </span>
+      )
+    },
+    {
+      name: i18n.translate('xpack.monitoring.kibana.listing.memorySizeColumnTitle', {
+        defaultMessage: 'Memory Size'
+      }),
+      field: 'process.memory.resident_set_size_in_bytes',
+      render: value => (
+        <span>
+          {formatNumber(value, 'byte')}
+        </span>
+      )
+    },
+    {
+      name: i18n.translate('xpack.monitoring.kibana.listing.requestsColumnTitle', {
+        defaultMessage: 'Requests'
+      }),
+      field: 'requests.total',
+      render: value => (
+        <span>
+          {formatNumber(value, 'int_commas')}
+        </span>
+      )
+    },
+    {
+      name: i18n.translate('xpack.monitoring.kibana.listing.responseTimeColumnTitle', {
+        defaultMessage: 'Response Times'
+      }),
+      // It is possible this does not exist through MB collection
+      field: 'response_times.average',
+      render: (value, kibana) => {
+        if (!value) {
+          return null;
+        }
+
+        return (
+          <div>
+            <div className="monTableCell__splitNumber">
+              { value && (formatNumber(value, 'int_commas') + ' ms avg') }
+            </div>
+            <div className="monTableCell__splitNumber">
+              { formatNumber(kibana.response_times.max, 'int_commas') } ms max
+            </div>
+          </div>
+        );
+      }
+    }
+  ];
+
+  return columns;
+};
+
+export class KibanaInstances extends PureComponent {
+  render() {
+    const {
+      instances,
+      clusterStatus,
+      angular,
+      setupMode,
+    } = this.props;
+
+    const dataFlattened = instances.map(item => ({
+      ...item,
+      name: item.kibana.name,
+      status: item.kibana.status,
+    }));
+
+    return (
+      <EuiPage>
+        <EuiPageBody>
+          <EuiPanel>
+            <ClusterStatus stats={clusterStatus} />
+          </EuiPanel>
+          <EuiSpacer size="m" />
+          <EuiPageContent>
+            <EuiMonitoringTable
+              className="kibanaInstancesTable"
+              rows={dataFlattened}
+              columns={getColumns(angular.kbnUrl, angular.$scope)}
+              sorting={this.sorting}
+              pagination={this.pagination}
```

Comment:
I think `this.sorting` and `this.pagination` will both be undefined since `KibanaInstances` dose not extend any classes that would have those as defaults.

If default `sorting` is not set you won't be able to sort on columns, and same if default `pagination` is not set it will hide the pagination footer